manifests/00-crd.yaml

```diff
@@ -209,7 +620,27 @@ spec:
                         privateKey:
                           description: PrivateKey points to secret containing the
                             private key, provided by AWS.
+                          properties:
+                            key:
+                              description: The key of the secret to select from.  Must
+                                be a valid secret key.
+                              type: string
+                            name:
+                              description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                                TODO: Add other useful fields. apiVersion, kind, uid?'
```

Comment:
Need to clean up our doc comments - this will show up in `oc explain`

Reply:
This is a part of `k8s.io/api/core/v1`.

Reply:
Something that we can do upstream!

Reply:
(obviously won't address in this PR)